### PR TITLE
chore(deps): setting up rtl and updating packages for future pr

### PR DESCRIPTION
**Pull request by Fewwy**
# Description
I am separating my work on the RTL into smaller pieces so it would be easier to review and merge

### CODE REVIEW

File: config/rtlwrapper.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { Provider } from 'react-redux';
+
+export const mountWithRouter = (Component) => {
+    return render(<BrowserRouter>{Component}</BrowserRouter>);
+};
+
+export const mountWithRouterAndProvider = (Component, store) => {
+    return render(
+        <Provider store={store}>
+            <BrowserRouter>{Component}</BrowserRouter>
```
Comment (mkholjuraev) [inline config/rtlwrapper.js:12]: Wondering if MemoryRouter would be better to use?

**Review by mkholjuraev — APPROVED** [review]
LGTM!

File: config/rtlwrapper.js (hunk 1 above)
Comment (Fewwy) [reply config/rtlwrapper.js:12]: I tried it and it didn't work :(

**Pull request merged**